front/src/front/static/js/Cookies.js

```diff
@@ -0,0 +1,23 @@
+export class Cookies {
+
+    static get(name) {
+        let cookies = document.cookie.split(';');
+        for (let i = 0; i < cookies.length; i++) {
+            let c = cookies[i].trim();
+            if (c.indexOf(name) === 0) {
```

Comment:
Nothing important, but consider using `c.startsWith(name)` as it will avoid iterating through the whole string for no reason